coordinator/src/main/scala/filodb.coordinator/queryplanner/SingleClusterPlanner.scala

```diff
@@ -225,11 +225,89 @@ class SingleClusterPlanner(dsRef: DatasetRef,
       case lp: ScalarFixedDoublePlan       => materializeFixedScalar(qContext, lp)
       case lp: ApplyAbsentFunction         => materializeAbsentFunction(qContext, lp)
       case lp: ScalarBinaryOperation       => materializeScalarBinaryOperation(qContext, lp)
+      case lp: SubqueryWithWindowing       => materializeSubqueryWithWindowing(qContext, lp)
+      case lp: TopLevelSubquery            => materializeTopLevelSubquery(qContext, lp)
```

Comment:
We need unit tests for physical plan materialization for different types of queries (a) top level (b) one level (c) nested. One concern I have is to make sure that we have selected the right starting point for leaf plan raw data.

Reply:
which spec would it be?

Reply:
SingleClusterPlannerSpec

Reply:
will consult with the cases in subquery-parsing branch of Vish' repository

Reply:
added tests